airflow/bin/cli.py

```diff
@@ -859,10 +859,12 @@ def serve_logs(filename):  # noqa
             mimetype="application/json",
             as_attachment=False)
 
+    WORKER_LOG_SERVER_BIND_IP = \
+        int(conf.get('celery', 'WORKER_LOG_SERVER_BIND_IP'))
```

Comment:
An IP is not an `int`, if I'm not mistaking.

Reply:
That's for sure... Thanks for noticing, jsut fixed it.